src/main/java/com/acoustic/salarycalculator/service/JobTitleServiceImpl.java

```diff
@@ -1,39 +1,39 @@
-package org.example.enumApproch.service;
+package com.acoustic.salarycalculator.service;
 
 import java.math.BigDecimal;
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 
-import org.example.enumApproch.dbConnection.DatabaseConnection;
-import org.example.enumApproch.enums.SalaryCalculatorEnum;
+import com.acoustic.salarycalculator.calculator.SalaryCalculator;
+import com.acoustic.salarycalculator.dbconnection.DatabaseConnection;
 
 
 public class JobTitleServiceImpl implements JobTitleService {
 
-    private final DatabaseConnection hikariCPDataSource;
+    private final DatabaseConnection databaseConnection;
     private final Connection connection;
 
```

Comment:
you are catching SQL exception and rethrowing more generic, this is not a good practice, in general we should throw as much specific exception as we can, here we could just print error message instead of runtimeexception, in other methods we print stack tace which isn't good for end user this should be a nice error message like "Connection to the db failed, please check your setup is correct ." etc..

Reply:
fixed